src/game/bg_public.h

```diff
@@ -2128,7 +2128,8 @@ void BG_AddPredictableEventToPlayerstate(int newEvent, int eventParm, playerStat
 
 void BG_PlayerStateToEntityState(playerState_t *ps, entityState_t *s, int time, qboolean snap);
 
-qboolean BG_PlayerTouchesItem(playerState_t *ps, entityState_t *item, int atTime);
+qboolean BG_PlayerTouchesItemNew(playerState_t *ps, entityState_t *item, int atTime);
+qboolean BG_PlayerTouchesItemOld(playerState_t *ps, entityState_t *item, int atTime);
```

Comment:
Perhaps a more generic / specific name would be welcome. 
Such as BG_PlayerTouchesItemSquare/CylinderBBox